tests/run-make/min-global-align/rmake.rs

```diff
@@ -0,0 +1,27 @@
+// This test checks that global variables respect the target minimum alignment.
+// The three bools `STATIC_BOOL`, `STATIC_MUT_BOOL`, and `CONST_BOOL` all have
+// type-alignment of 1, but some targets require greater global alignment.
+// See https://github.com/rust-lang/rust/pull/44440
+
+//@ only-linux
+// Reason: this test is target-independent, considering compilation is targeted
+// towards linux architectures only.
+
+use run_make_support::{assert_count_is, llvm_components_contain, rfs, rustc};
+
+fn main() {
+    // Most targets are happy with default alignment -- take i686 for example.
+    if llvm_components_contain("x86") {
+        rustc().target("i686-unknown-linux-gnu").emit("llvm-ir").input("min_global_align.rs").run();
+        assert_count_is(3, rfs::read_to_string("min_global_align.ll"), "align 1");
+    }
```

Comment:
Remark: I wonder if we *could* try to support revisions in rmake.rs. In ui tests and codegen tests `revisions` also define corresponding `cfg`s, maybe we could use them here as well (this remark is for myself).

Something like

```
//@ revisions: i686 systemz
//@[i686] needs-llvm-components: x86
//@[systemz] needs-llvm-components: systemz

fn main() {
    #[cfg(i686)]
    {
        // i686 specific assertions
    }
    #[cfg(systemz)]
    {
        // systemz specific assertions
    }
}
```

Just a thought.